hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp

```diff
@@ -22,6 +22,28 @@
 <%@ include file="/WEB-INF/jsp/include.jsp"%>
 <%@ include file="/WEB-INF/jsp/header.jsp"%>
 
+
+<sec:authorize
+  ifAnyGranted="ROLE_REDIRECT_TO_HRS_FLUID_TIME">
+  <c:choose>
+    <c:when test="${not empty hrsUrls['Fluid Time']}">
+      <script>
+        window.location.replace("${hrsUrls['Fluid Time']}");
+      </script>
+    </c:when>
+    <c:otherwise>
+       <p>Error. Try again later or contact the Help Desk.</p>
+       <p>Technical details: you are in MyUW role ROLE_DIRECT_TO_HRS_FLUID_TIME,
```

Comment:
Missing the RE in REDIRECT

Reply:
Excellent catch, but now I need to worry about where I copy and pasted that from if anywhere...